src/cogs/fun.py

```diff
@@ -0,0 +1,143 @@
+                tests_per_one_million = json_stats["testsPerOneMillion"]
+
+                embed2 = discord.Embed(title=f"**COVID-19 Status Of {country}**!",
+                                       description="This Information Isn't Live Always, Hence It May Not Be Accurate!",
+                                       colour=0x0000ff, timestamp=ctx.message.created_at)
+                embed2.add_field(name="**Total Cases**", value=f'{total_cases:,}' if total_cases is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Today Cases**", value=f'{today_cases:,}' if today_cases is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Total Deaths**",
+                                 value=f'{total_deaths:,}' if total_deaths is not None else "Null", inline=True)
+                embed2.add_field(name="**Today Deaths**",
+                                 value=f'{today_deaths:,}' if today_deaths is not None else "Null", inline=True)
+                embed2.add_field(name="**Recovered**", value=f'{recovered:,}' if recovered is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Active**", value=f'{active:,}' if active is not None else "Null", inline=True)
+                embed2.add_field(name="**Critical**", value=f'{critical:,}' if critical is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Cases Per One Million**",
+                                 value=f'{cases_per_one_million:,}' if cases_per_one_million is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Deaths Per One Million**",
+                                 value=f'{deaths_per_one_million:,}' if deaths_per_one_million is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Total Tests**", value=f'{total_tests:,}' if total_tests is not None else "Null",
+                                 inline=True)
+                embed2.add_field(name="**Tests Per One Million**",
+                                 value=f'{tests_per_one_million:,}' if tests_per_one_million is not None else "Null",
+                                 inline=True)
+
+                embed2.set_thumbnail(
+                    url="https://cdn.discordapp.com/attachments/564520348821749766/701422183217365052/2Q.png")
+                await ctx.send(embed=embed2)
+
+        except:
+            embed3 = discord.Embed(title="Invalid Country Name Try Again..!", colour=0xff0000,
+                                   timestamp=ctx.message.created_at)
+            embed3.set_author(name="Error!")
+            await ctx.send(embed=embed3)
+
+    @commands.command()
+    async def ping(self, ctx):
+        """Pong! Get the bot's response time"""
+        await ctx.channel.purge(limit=1)
```

Comment:
What's this for?

Reply:
This was just to remove the ping command, its not really necessary if we want to leave the commands visible.